src/ontology/cl-edit.owl

```diff
@@ -3533,9 +3533,9 @@ SubClassOf(obo:CL_0000032 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000114))
 
 # Class: obo:CL_0000033 (apocrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "ISBN:0721662544"^^xsd:string) obo:IAO_0000115 obo:CL_0000033 "An apocrine cell is a secretory cell characterized loss of part of the cytoplasm during the process of secretion - secretions are budded off through the plasma membrane producing membrane bound vesicles containing the secreted substance."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "ISBN:0721662544"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:25960390"^^xsd:string) obo:IAO_0000115 obo:CL_0000033 "An apocrine cell is an exocrine cell characterized by loss of part of the cytoplasm during the process of secretion. The secreted substance is accumulated at the apical end and budded off through the plasma membrane producing secreted vesicles."^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000033 "apocrine cell"^^xsd:string)
-SubClassOf(obo:CL_0000033 obo:CL_0000151)
+SubClassOf(obo:CL_0000033 obo:CL_0000152)
```

Comment:
Agree with this change. Since remodelling the term was not part of the original ticket, I recommend updating the issue description and issue title. This pull request is no longer just fixing a typo. 
By updating the description and title, it will facilitate users in finding the origin of changes in the ontology and also be reflected in the release notes.